tests/test_http.py

```diff
@@ -116,6 +122,35 @@ def client_with_url(timeout=3000):
                        url="http://127.0.0.1:6080", timeout=timeout)
 
 
+def client_context_with_header_factory(timeout=3000):
+    return client_context(addressbook.AddressBookService,
+                          url="http://127.0.0.1:6080",
+                          timeout=timeout,
+                          http_header_factory=THttpHeaderFactory())
+
+
+def client_context_with_custom_header_factory(timeout=3000):
+    return client_context(addressbook.AddressBookService,
+                          url="http://127.0.0.1:6080",
+                          timeout=timeout,
+                          http_header_factory=CustomHeaderFactory())
+
+
+def client_with_header_factory(timeout=3000):
+    return make_client(addressbook.AddressBookService,
+                       url="http://127.0.0.1:6080",
+                       timeout=timeout,
+                       http_header_factory=THttpHeaderFactory(
+                           {"X-REQUEST-ID": str(uuid.uuid4())}))
+
+
+def client_with_custom_header_factory(timeout=3000):
+    return make_client(addressbook.AddressBookService,
+                       url="http://127.0.0.1:6080",
+                       timeout=timeout,
+                       http_header_factory=CustomHeaderFactory)
```

Comment:
Looks like we are missing a parenthese here.

Reply:
fixed